src/main/java/networkbook/model/person/Person.java

```diff
@@ -25,22 +25,28 @@ public class Person implements Identifiable<Person> {
     private final UniqueList<Email> emails;
 
     // Data fields
-    private final Address address;
+    private final WebLink webLink;
+    private final GraduatingYear graduatingYear;
+    private final Course course;
+    private final Specialisation specialisation;
```

Comment:
Take note that some of these fields are lists instead of a single entry. People assigned to tasks please make changes accordingly once this pull request is merged.